docs/flink.md

```diff
@@ -0,0 +1,15 @@
+# Apache Flink
+
+## Logs
+
+Flink logs contain the following fields in the [`LogEntry`](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry) for Client, Jobmanager and Taskmanagers:
+
+| Field | Type | Description |
+| ---   | ---- | ----------- |
+| `jsonPayload.source` | string | Module and/or thread  where the log originated |
+| `jsonPayload.logger` | string | Name of the logger where the log originated |
```

Comment:
I don't see this field in the expected logs or in the parse regex. Are we missing something?

Reply:
mb. removed `jsonPayload.logger`